packages/PHPUnit/tests/Rector/MethodCall/ExplicitPhpErrorApiRector/Fixture/php_combination.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\MethodCall\ExplicitPhpErrorApiRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class RefactorCombination extends TestCase
+{
+    public function test()
+    {
+        $this->expectException(\PHPUnit\Framework\TestCase\Deprecated::class);
+        $this->expectException(\PHPUnit\Framework\TestCase\Error::class);
+        $this->expectException(\PHPUnit\Framework\TestCase\Notice::class);
+        $this->expectException(\PHPUnit\Framework\TestCase\Warning::class);
+    }
+}
+
```

Comment:
This class duplicates the other 4 cases right?
What's the added value?

Reply:
fixed that